File: quasar/Quasar.DataTransfer/Vendas/Request/ItemVendaInserirRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Quasar.DataTransfer.Vendas.Request
{
    public class ItemVendaInserirRequest
    {
        public int Quantidade {get; set;}
        public int CodProduto {get; set;}
    }
}

File: quasar-front/src/app/vendas/models/requests/itemVendaInserir.request.ts
export class ItemVendaInserirRequest {
  public quantidade: number;
  public codProduto: number;

  constructor(params: Partial<ItemVendaInserirRequest>) {
    this.quantidade = params.quantidade || 0;
    this.codProduto =  params.codProduto || 0;
  }
}
